app/code/local/Edge/PrintProof/controllers/ProofController.php

```diff
@@ -39,8 +39,15 @@ public function addToExistingAction()
 
     public function approveAction()
     {
+        $proofId = $this->getRequest()->getParam('proof_id', false);
+
+        Mage::helper('printproof')->addToExisting([
+            'proof_id' => $proofId,
+            'comment' => 'Approved by customer.'
```

Comment:
Is other controller who deals with that action and will save the exact comment the admin writes.
If admin approve or reject he is able to add the comment.
If user reject is able to add a comment, if user approved the message will be "Approved by customer"